generators/server/templates/_README.md

```diff
@@ -126,7 +126,7 @@ To optimize the <%= baseName %> application for production, run:
     ./gradlew -Pprod clean bootRepackage<% } %>
 
 <%_ if(!skipClient) { _%>
-This will concatenate and minify the client CSS and JavaScript files. It will also modify `index.html` so it references these new files.
+This will concatenate and minify the client CSS and JavaScript files. It will also modify [index.html](index.html) so it references these new files.
```

Comment:
This link doesn't exist. It should be `src/main/webapp/index.html` instead
Can you do another PR with [ci skip] in your commit message? So it won't trigger a new Travis build :construction_worker_man: :construction_worker_woman: